feature/gribi/ate_tests/base_hierarchical_nhg_update/base_hierarchical_nhg_update_test.go

```diff
@@ -233,10 +233,10 @@ func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
 		Name:    ygot.String(vrfName),
 		Enabled: ygot.Bool(true),
 		Type:    telemetry.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L3VRF,
-		EnabledAddressFamilies: []telemetry.E_Types_ADDRESS_FAMILY{
-			telemetry.Types_ADDRESS_FAMILY_IPV4,
-			telemetry.Types_ADDRESS_FAMILY_IPV6,
-		},
+		//EnabledAddressFamilies: []telemetry.E_Types_ADDRESS_FAMILY{
```

Comment:
Let's not just comment this out -- we should either explicitly remove it or keep it there.

Reply:
Addressed with the latest commit.